index.html

```diff
@@ -56,7 +56,8 @@ <h2>Privacy Protections Tests</h2>
 	<li><a href='./privacy-protections/referrer-trimming/'>Referrer trimming</a></li>
 	<li><a href='./privacy-protections/https-upgrades/'>HTTPS upgrades</a></li>
 	<li><a href='./privacy-protections/https-loop-protection/'>HTTPS upgrade loop protection</a></li>
-	<li><a href='./privacy-protections/click-to-load/'>Facebook click to load</a></li>
+	<li><a href='./privacy-protections/facebook-click-to-load/'>Facebook click to load</a></li>
```

Comment:
sorry, I'm afraid you can't change that URL because some integration tests (like in our extension) already depend on it. So our options are:

- make a redirect in server.js so that /click-to-load/ loads /facebook-click-to-load/
- keep facebook where it is now
- move youtube to  a subfolder in /click-to-load/ ? Not sure about this one, but maybe it makes sense?

Reply:
Oops, I didn't realise that sorry. I've moved it back for now, we can always rethink if it becomes problematic.